Source/WebCore/page/Navigation.h

```diff
@@ -156,7 +156,7 @@ class Navigation final : public RefCounted<Navigation>, public EventTarget, publ
 
     void abortOngoingNavigationIfNeeded();
 
-    std::optional<Ref<NavigationHistoryEntry>> findEntryByKey(const String& key);
+    NavigationHistoryEntry* findEntryByKey(const String&) const;
```

Comment:
I think it would be nice if we could keep this as returning `std::optional<RefNavigationHistoryEntry>>`